test/integration/Service.spec.ts

```diff
@@ -35,10 +35,10 @@ describe('API Service', () => {
       },
       lnd: {
         LTC: {
-          disable: true,
+          disable: false,
         },
         BTC: {
-          disable: true,
+          disable: false,
```

Comment:
Why are these values changed?

Reply:
With the new logic querying for a `swapClient` will throw if it's not found. The service spec relies on the the swap client to be available. With the simulation tests in place I'm beginning to question the need to have these kinds of acceptance tests 🤔 .